node_modules/oae-core/tenantlandingpage/css/tenantlandingpage.css

```diff
@@ -0,0 +1,154 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#tenantlandingpage {
+    font-weight: 200;
+    margin: 0 auto;
+    max-width: 985px;
+}
+
+#tenantlandingpage .tenantlandingpage-row {
+    display: table;
+    margin: 0;
+    width: 100%;
+}
+
+/************
+ ** Blocks **
+ ************/
+
+#tenantlandingpage .tenantlandingpage-block {
+    background-position: center;
+    background-size: cover;
+    display: table-cell;
+    float: none;
+    padding: 25px 25px;
+    vertical-align: top;
+}
+
+#tenantlandingpage .tenantlandingpage-row:first-child .tenantlandingpage-block {
+    padding-top: 0;
+}
+
+/* Search block */
+
+#tenantlandingpage .tenantlandingpage-block-search-container:only-child {
```

Comment:
The removed block 

```
#index-page-container .index-block-search-container {
    padding: 35px 20px;
}
```

is still necessary in case a search block is configured on a row different than the first one or when there are other blocks on the same line